spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/audit/service/DefaultAuditRecordService.java

```diff
@@ -39,23 +43,34 @@
  */
 public class DefaultAuditRecordService implements AuditRecordService {
 
+	public static final String STREAM_DEFINITION_DSL_TEXT = "streamDefinitionDslText";
```

Comment:
It would be better to move all these constants into `AuditRecordService` as that would avoid having these constants recreated again in some other classes.

Reply:
This is going to happen as part of larger refactoring https://github.com/spring-cloud/spring-cloud-dataflow/issues/2477

Reply:
Yes - will be done as part of https://github.com/spring-cloud/spring-cloud-dataflow/issues/2477 I wanted to do it as part of this PR but as it got a big bigger...